app/presenters/audit/deploy_presenter.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+class Audit::DeployPresenter
+  ## Deploy presenter for Audit Logger
+  ## Returns deploy object with only id, stage id and name, reference, deployer and buddy id and name, and time
```

Comment:
detailed field description is kind of redundant / will get out of date quickly ...